addons/new-osio-addon/pom.xml

```diff
@@ -25,6 +25,11 @@
     </dependency>
 
     <!-- Test dependencies -->
+    <dependency>
+      <groupId>io.fabric8.launcher</groupId>
+      <artifactId>launcher-common-test</artifactId>
```

Comment:
launcher-common-test? But that only contains Hoverfly classes. Perhaps call it `launcher-hoverfly`?